config/initializers/content_security_policy.rb

```diff
@@ -4,23 +4,82 @@
 # For further information see the following documentation
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
-Rails.application.configure do
-  config.content_security_policy do |policy|
-    policy.default_src :self, :https
-    policy.font_src    :self, :https, :data
-    policy.img_src     :self, :https, :data
-    policy.object_src  :none
-    policy.script_src  :self, :https
-    policy.style_src   :self, :https
-    # Specify URI for violation reports
-    # policy.report_uri "/csp-violation-report-endpoint"
-  end
+# See https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP for more CSP info.
```

Comment:
Really appreciate the docs and annotations 👍🏼 
Can you make sure they are `yardoc` compatible, that could mean changing "See" to `@see` for example.

Reply:
Updated code block now uses yardoc specific docs and annotations